time-to-live.md

```diff
@@ -141,6 +136,8 @@ ALTER TABLE orders TTL_JOB_INTERVAL = '24h';
 
 `TTL_JOB_INTERVAL` 的默认值是 `1h`。
 
+在执行 TTL 任务时，TiDB 会将表拆分为最多 64 份。这些子任务将以 `Region` 为最小单位，在多个 TiDB 实例中并行执行。你可以通过设置系统变量 [`tidb_ttl_running_tasks`](/system-variables.md#tidb_ttl_running_tasks-从-v700-版本开始引入) 来限制整个集群中同时执行的 TTL 子任务数量。然而，并非所有表的 TTL 任务都可以被拆分为子任务。请参考 [使用限制](#使用限制) 以了解哪些表的 TTL 任务不能被拆分。
```

Comment:
I added a new description for the TTL tasks and the limitations. PTAL @lcwangchao @ran-huang